### PR TITLE
Added client model

### PR DESCRIPTION
**Pull request by domenic99**
#9 Client model created.

### CODE REVIEW

File: db/migrate/20161206115524_create_clients.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreateClients < ActiveRecord::Migration[5.0]
+  def change
+    create_table :clients do |t|
+      t.string :email
+      t.string :phone
```
Comment (rakvium) [inline db/migrate/20161206115524_create_clients.rb]: @domenic99 are you sure that we could have any clients without a phone?

Comment (domenic99) [inline db/migrate/20161206115524_create_clients.rb]: @rakvium No. My fail )

**Comment by domenic99** [comment]
Changed

File: test/fixtures/clients.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  email: MyString
+  phone: MyString
+
+two:
+  email: MyString
+  phone: MyString
```
Comment (rakvium) [inline test/fixtures/clients.yml]: This fixture won't work accordingly to https://github.com/rakvium/taxi-api/pull/15/files#diff-0a8fa39aebf1877d55271c814b471dadR10

**Comment by domenic99** [comment]
Fix fir fixtures

**Comment by rakvium** [comment]
@domenic99 please fix conflicts.

**Comment by domenic99** [comment]
@rakvium Check out my fix )

**Pull request merged**